### PR TITLE
chore(docs): remove leading dollar sign

### PR DESCRIPTION
**Pull request by nhedger**
See https://github.com/reactphp/http/pull/458

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -605,13 +605,13 @@ To run the test suite, you first need to clone this repo and then install all
...
 ```bash
-$ composer install
+composer install
 ```
 
 To run the test suite, go to the project root and run:
 
 ```bash
-$ php vendor/bin/phpunit
+php vendor/bin/phpunit
```
Comment (clue) [inline README.md]: Looks like a good opportunity to also get rid of the `php` binary here for consistency reasons (not needed and also not used in most other components) :+1:

**Comment by clue** [comment]
The build error is unrelated to this changeset and is actually caused by an upstream regression in PHP 8.1.7 and PHP 8.0.20 only (see https://github.com/php/php-src/issues/8575, https://github.com/php/php-src/pull/8571 and https://github.com/php/php-src/issues/8827). There's ongoing effort to revert this for upcoming PHP versions again, so the build should be green again by then, no changes required here.

**Comment by clue** [comment]
> The build error is unrelated to this changeset and is actually caused by an upstream regression in PHP 8.1.7 and PHP 8.0.20 only (see [php/php-src#8575](https://github.com/php/php-src/issues/8575), [php/php-src#8571](https://github.com/php/php-src/pull/8571) and [php/php-src#8827](https://github.com/php/php-src/issues/8827)). There's ongoing effort to revert this for upcoming PHP versions again, so the build should be green again by then, no changes required here.

See #96 for temporary test adjustments to skip affected tests.

**Review by clue — APPROVED** [review]
@nhedger Thanks for the quick update, changes LGTM! :+1:

**Review by WyriHaximus — APPROVED** [review]

**Pull request merged**